explorer/site-config/anvil/images/favicons/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
```

Comment:
@BruceRodrigues is there a way we can keep from having multiple of these types of files? Can can we keep a single copy in a config/common directory or similar and copy that in as well to /public